package.json

```diff
@@ -58,7 +59,7 @@
     "prettier": "2.2.1",
     "reflect-metadata": "0.1.13",
     "rxjs": "^7.2.0",
-    "typescript": "4.2.4"
+    "typescript": "^5.4.5"
```

Comment:
Does it change anything in terms of your change?

Reply:
This lets us use a `symbol` as a property key, so it doesn't show up in the interface.